src/app/ngrx/effects/cms.effects.ts

```diff
@@ -0,0 +1,65 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/concatMap';
+import 'rxjs/add/operator/switchMap';
+import 'rxjs/add/operator/map';
+import { Action, Store } from '@ngrx/store';
+import { Actions, Effect } from '@ngrx/effects';
+import { selectPodcasts } from '../reducers';
+import { PodcastModel, EpisodeModel } from '../model';
+import { CmsPodcastsSuccessAction, CmsPodcastsFailureAction, ActionTypes } from '../actions';
+import { CmsService, HalDoc } from '../../core';
+
+@Injectable()
+export class CmsEffects {
+  podcasts: PodcastModel[] = [];
+
+  @Effect()
+  loadPodcasts$: Observable<Action> = this.actions$
+    .ofType(ActionTypes.CMS_PODCASTS)
+    .switchMap(() => {
+      // would be great if this worked
+      //  * would simplify app.component to just calling this.store.dispatch(new CmsPodcastsAction());
+      //  * would provide state management for loading and errors
+      // the followItems gets called, but it never returns, no errors on the console or caught exceptions, possibly swallowed error
+      // reason this isnt working is that auth is an observable, need to follow
+      return this.cms.auth.followItems('prx:series', {per: this.cms.auth.count('prx:series'), filters: 'v4', zoom: 'prx:distributions'})
+        .flatMap((docs: HalDoc[]) => {
+          const podcasts: PodcastModel[] = docs.map(doc => {
+            return {
+              doc,
+              seriesId: doc['id'],
+              title: doc['title']
+            };
+          });
+          const dist$ = podcasts.map(p => this.getPodcastDistribution(p));
+          return Observable.forkJoin(...dist$).map(() => new CmsPodcastsSuccessAction({podcasts}));
```

Comment:
I really hoped that was the case, and I thought the direct import would fix that forkJoin problem. I've tried new styleguide builds with this as well and still can't seem to get it working. I checked it into this branch so I could come back to it later though because this would be nice to have. I really appreciate you looking at it!